src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -38,7 +38,11 @@
 
 #include "utils/logger.h"
 #include "utils/date_time.h"
+#if __QNX__
+#include "utils/qdb_wrapper/sql_database.h"
```

Comment:
@BSolonenko please try to resolve this issue with cmakes

Reply:
@AKalinich-Luxoft Fixed in b2b51c312fc57007af9a6626c0e88bfe2f0b594e